gbasis/evals/density.py

```diff
@@ -110,6 +110,7 @@ def evaluate_deriv_reduced_density_matrix(
     points,
     transform=None,
     coord_type="spherical",
+    deriv_type="general"
```

Comment:
I don't know if `black` is part of QA, but I think it would complain about not having the comma at the end.

Reply:
It didn't show up using `tox -e qa` but maybe I missed it . My reasoning is that it is the last argument given to the function?